platform/viewer/src/components/ViewportGrid.tsx

```diff
@@ -42,18 +42,23 @@ function ViewerViewportGrid(props) {
           continue;
         }
 
+        // console.log("Applying HP matchDetails", i, matchDetails[i]);
```

Comment:
commented console.log (I haven't commented on all of these, but there are at least 20 unnecessary line additions)